pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -159,6 +159,8 @@ func (m *MigCluster) BuildRestConfig(c k8sclient.Client) (*rest.Config, error) {
 		Host:            m.Spec.URL,
 		BearerToken:     string(secret.Data[SaToken]),
 		TLSClientConfig: tlsClientConfig,
+		Burst:           1000,
```

Comment:
Where did these burst/QPS values come from? Did we do some testing?

Reply:
What is the observed effect of these new values vs. defaults?

Reply:
@jortel do you know where these values came from? I copied them from the non-admin branched after you pointed me to them.

As for the observed effect, it causes the queries to take a fraction of the time they do without due to elimination of rate limiting.

Reply:
For reference: https://github.com/konveyor/mig-controller/blob/non-admin/pkg/apis/migration/v1alpha1/migcluster_types.go#L174

Reply:
```

Reply:
```
The number of client requests or API calls that are sent to the API server is determined by the Queries per second (QPS) value and the number of concurrent requests that can be processed by the API server is determined by the maxRequestsInFlight setting. The number of requests the client can make in excess of the QPS rate depends on the burst value, this is helpful for applications that are bursty in nature and can make irregular number of requests. 
```
https://docs.openshift.com/container-platform/3.11/scaling_performance/host_practices.html